internals/scripts/helpers/config.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const merge = require('lodash.merge');
+const path = require('path');
+
+const baseConfig = require('../../../app.base.json');
+const extendedConfigFile = path.join(__dirname, '..', '..', '..', 'app.json');
+const extendedConfig = fs.existsSync(extendedConfigFile) ? require(extendedConfigFile) : {};
+const config = merge({}, baseConfig, extendedConfig);
+
+const placeholders = {
+  $SIGNALS_ANDROID_ICON: config.head.androidIcon,
+  $SIGNALS_BACKGROUND_COLOR: config.head.backgroundColor,
+  $SIGNALS_CONFIG: JSON.stringify(config),
+  $SIGNALS_FAVICON: config.head.favicon,
+  $SIGNALS_IOS_ICON: config.head.iosIcon,
+  $SIGNALS_MATOMO_SITE_ID: config.matomo.siteId,
+  $SIGNALS_MATOMO_URL_BASE: config.matomo.urlBase,
+  $SIGNALS_PWA_SHORT_TITLE: config.language.shortTitle,
+  $SIGNALS_PWA_TITLE: config.language.title,
+  $SIGNALS_SITE_TITLE: config.language.siteTitle,
+  $SIGNALS_STATUS_BAR_STYLE: config.head.statusBarStyle,
+  $SIGNALS_THEME_COLOR: config.head.themeColor,
```

Comment:
Since these props are no longer env vars, we don't have to prefix them with `$`, do we?

Reply:
They've never been env vars. The `$` is a nice way to avoid any conflicts since it searches and replaces all of these strings in `index.html`.

Reply:
They were, as per the contents of `start.sh`.

Reply:
You are right indeed!